### DISCUSSION TITLE
No Volume Controls - v 2.0.0b140

### DISCUSSION THREAD

**Discussion by mllockwood** [music-assistant/support#2153] [Q&A] [answered]
After upgrading to version 2.0.0b140, all volume controls are missing from the UI.  The slider at bottom right is gone for active playing song, and none of the individual volume sliders are appearing when viewing individual speakers.

![image](https://github.com/music-assistant/hass-music-assistant/assets/42590777/0b6c539b-2c16-494a-9319-76c49e5425c7)

![image](https://github.com/music-assistant/hass-music-assistant/assets/42590777/f1e17f69-3781-464a-bf02-d50f8ffb2970)

**Comment by mllockwood** [comment]
Also, when the Player List popup is displayed, I am unable to close it by clicking the X.  I have to refresh the MA page to get it to close.

**Comment by marcelveldt** [comment, marked as answer]
A Vue (our frontend framework) update broke our UI. We published b141 with a hotfix

**Reply by mllockwood** [reply]
That fixed it.  Thank you and all the developers for all your work on this project!!!
